apps/core/components/ItineraryCard/itineraryDetail/ItineraryDetailWrapper.native.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+import { Modal } from '@kiwicom/margarita-components';
```

Comment:
Don't we have a modal from `universal-components` that is better?

Reply:
The margarita version is now based on `universal-components` one, so it is also better.
Margarita version adds that extra white background with rounded corners and center (web) bottom (mobile) content alignment. As mentioned in this PR https://github.com/kiwicom/margarita/pull/337

Reply:
Aha 👍